lib/salus/scanners/brakeman.rb

```diff
@@ -110,6 +110,18 @@ def merged_ignore_file_contents
     def ignore_list
       return [] unless user_supplied_ignore?
 
+      if !@config['ignore'].is_a?(String)
+        msg = "Brakeman ignore path #{@config['ignore']} is a/an #{@config['ignore'].class} and"\
+        " not a String. "
+        report_error(msg)
+        return []
+      end
+
+      if !File.exist?(@config['ignore'])
+        msg = "Brakeman ignore path #{@config['ignore']} does not exist"
```

Comment:
Please test by build local container and run salus.  
Try to use a valid brakeman ignore path and the code will still report the error.

Reply:
You probably need a `File.join` like here https://github.com/coinbase/salus/blob/2f3dbcba104c83dcd8dee178b97b8075d3c44e6f/lib/salus/scanners/brakeman.rb#L235

Reply:
It would be great to add a test case too.